ssl/statem/extensions_clnt.c

```diff
@@ -530,23 +530,8 @@ EXT_RETURN tls_construct_ctos_supported_versions(SSL *s, WPACKET *pkt,
         return EXT_RETURN_FAIL;
     }
 
-    /*
-     * TODO(TLS1.3): There is some discussion on the TLS list as to whether
-     * we should include versions <TLS1.2. For the moment we do. To be
-     * reviewed later.
```

Comment:
(The final text is quite explicit that clients MUST send supported_versions containing all versions of TLS which chey are prepared to negotiate.)